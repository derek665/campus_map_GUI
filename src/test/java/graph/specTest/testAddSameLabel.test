# test when an edge with same label, parent, child is added

# Create graph
CreateGraph dupl2

# Add 2 nodes
AddNode dupl2 n1
AddNode dupl2 n2

# Add 1 edge
AddEdge dupl2 n1 n2 e1

# Add the same edge again
AddEdge dupl2 n1 n2 e1

# Print children of n1
ListChildren dupl2 n1