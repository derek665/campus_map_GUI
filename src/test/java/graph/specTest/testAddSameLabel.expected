# test when an edge with same label, parent, child is added

# Create graph
created graph dupl2

# Add 2 nodes
added node n1 to dupl2
added node n2 to dupl2

# Add 1 edge
added edge e1 from n1 to n2 in dupl2

# Add the same edge again
# no change since e1 already exists

# Print children of n1
the children of n1 in dupl2 are: n2(e1)